DuckDuckGo.xcodeproj/project.pbxproj

```diff
@@ -1220,7 +1221,6 @@
 		317045BF2858C6B90016ED1F /* AutofillInterfaceEmailTruncatorTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AutofillInterfaceEmailTruncatorTests.swift; sourceTree = "<group>"; };
 		31794BFF2821DFB600F18633 /* DuckUI */ = {isa = PBXFileReference; lastKnownFileType = wrapper; path = DuckUI; sourceTree = "<group>"; };
 		31951E8D2823003200CAF535 /* AutofillLoginDetailsHeaderView.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = AutofillLoginDetailsHeaderView.swift; sourceTree = "<group>"; };
-		31951E94282310FF00CAF535 /* WebsiteAccountExtension.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = WebsiteAccountExtension.swift; sourceTree = "<group>"; };
```

Comment:
This file doesn't appear to be deleted, was this intentional?

Reply:
Ah sorry for confusion here - that file was moved to BSK as part of https://github.com/duckduckgo/iOS/pull/1911 but it wasn't removed from the .pbxproj so has been hanging around like a bad smell and I just couldn't take it anymore 😅   
![image](https://github.com/duckduckgo/iOS/assets/91189922/9beb9fe8-7808-449b-9b53-fca74e9b5a90)